cypress/support/ui_commands.js

```diff
@@ -388,3 +388,13 @@ Cypress.Commands.add('updateChannelHeader', (text) => {
         type('{enter}').
         wait(TIMEOUTS.TINY);
 });
+
+// ************************************************************
+// Cookie
+// ************************************************************
+
+Cypress.Commands.add('clearWhiteListedCookies', () => {
```

Comment:
Should this just be a part of `apiLogout`? I think we'd want to do it everytime we logout, but maybe not.

Reply:
Sure, that makes sense, I'll update.